samples/java_springboot/02.echo-bot/README.md

```diff
@@ -15,26 +17,26 @@ This sample is a Spring Boot app and uses the Azure CLI and azure-webapp Maven p
   - Build the sample using `mvn package`
   - Run it by using `java -jar .\target\bot-echo-sample.jar`
 
-- Test the bot using Bot Framework Emulator
-
-  [Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+## Testing the bot using Bot Framework Emulator
+[Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
 
   - Install the Bot Framework Emulator version 4.3.0 or greater from [here](https://github.com/Microsoft/BotFramework-Emulator/releases)
```

Comment:
Check the [sample 16](https://github.com/microsoft/BotBuilder-Samples/tree/Java-R13/samples/java_springboot/16.proactive-messages) for this. We are not indicating to install 4.3.0 or greater